### PR TITLE
[docs] Update concurrencies info in FAQ

### PR DESCRIPTION
**Pull request by sjchmiela**
# Why

Soon we'll offer this in a self-checkout manner.

# How

Updated information with links and stuff.

# Test Plan

Clicked the links, they got me to where I wanted them to.

# Checklist

- [x] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [x] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [x] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/billing/faq.mdx [hunk 1]
```diff
@@ -112,4 +112,8 @@ To view the cost of a large build, go to [Billing](https://expo.dev/settings/bil
 
 ### How do I increase build concurrencies on my account?
 
-[Contact us](https://expo.dev/contact) for more information.
+If you're already subscribed to a paid EAS plan, you can buy additional concurrencies in [the Add-ons section on the Billing](https://expo.dev/settings/billing#addons) page.
```
Comment (amandeepmittal) [inline docs/pages/billing/faq.mdx:115]:
```suggestion
If you're already subscribed to a paid EAS plan, you can buy additional concurrencies in the [**Add-ons** section](https://expo.dev/settings/billing#addons) under **Billing**.
```

Let's make settings/pages name bold as this is the pattern we follow in this doc for describing paths to other settings/areas of billing page.

File: docs/pages/billing/faq.mdx [hunk 2]
```diff
@@ -112,4 +112,8 @@ To view the cost of a large build, go to [Billing](https://expo.dev/settings/bil
...
+
+If you're still on the Free plan, you will need to set up a paid subscription. [Click here](https://expo.dev/accounts/[account]/settings/billing/cart) to choose a plan. You can select a number of additional concurrencies to add to your subscription on the checkout page.
```
Comment (amandeepmittal) [inline docs/pages/billing/faq.mdx:117]:
```suggestion
If you're on the Free plan, you'll need to set up a paid subscription. [Click here](https://expo.dev/accounts/[account]/settings/billing/cart) to choose a new plan. Then, select a number of additional concurrencies to add to your subscription on the checkout page.
```

"If you're still" sounds like a bit aggressive to me. I think we remove the word "still" here.

**Review by amandeepmittal — COMMENTED** [review]
A couple of minor suggestions.

**Review by jonsamp — APPROVED** [review]
I agree with Aman's suggestions. Once you change those, I think we should ship this.

**Review by amandeepmittal — APPROVED** [review]

**Pull request merged**